addons/sale/wizard/mass_cancel_quotations_wizard_views.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="mass_cancel_quotations_wizard_view_form" model="ir.ui.view">
+        <field name="name">sale.mass.cancel.quotations.form</field>
+        <field name="model">sale.mass.cancel.quotations</field>
+        <field name="arch" type="xml">
+            <form string="Cancel quotations">
+                <field name="sale_order_ids" invisible="1"/>
+                <field name="nb_sale_order" invisible="1"/>
+                <field name="has_confirmed_order" invisible="1"/>
+                <div attrs="{'invisible':[('has_confirmed_order', '=', False)]}">
+                    <p class="alert alert-warning fw-bold" role="alert">
+                        Some confirmed orders are selected. Their related documents might be affected
+                        by the cancellation.
+                    </p>
+                </div>
+                <div attrs="{'invisible': [('nb_sale_order', '>', 1)]}">
+                    Are you sure you want to cancel the selected quotation ?
+                </div>
+                <div attrs="{'invisible': [('nb_sale_order', '=', 1)]}">
+                    Are you sure you want to cancel the <field name="nb_sale_order"/> selected
+                    quotations ?
+                </div>
```

Comment:
`div t-if=blabla` and `div t-else=""` ?

Reply:
& same here, a space before the question mark.

Reply:
Thanks for the suggestion! At first, I thought it was a great idea but it turns out that wizards are not QWeb templates and therefore, not compatible with `t-if` and `t-else`. It could have cleared a bit this template though, too bad :cry: